README.md

```diff
@@ -164,6 +164,39 @@ descript
   });
 ```
 
+### Adding your own searcher
+
+You can add your own searcher
```

Comment:
@Helen-Mobify can you add some details about what this does please? It's not clear what a "searcher" is in this context.